README.md

```diff
@@ -1,11 +1,66 @@
-# jupiter-repo-dependencies
+# JupiterOne Repository Relationship Generator
 
-- Put the repos you want to run the script on in the `repos` folder.
-- Use `yarn deps` to run the script.
+A script that generates graphql relationships between a repository and its dependencies.
+This script will look for dependencies within a repository's `package.json` file as well as the `dependencies.yaml`file if the repository has a `deploy` directory.
 
-## Package Scope Input
+This script uses the JupiterOne graphql graph database.
 
-- You will be asked to input as many package scopes as you like.
-- For example, if you have packages that follow this pattern: @package/*, then you would input @package.
+## Before Running
+- Create a directory containing the repositories that you would like the script to be run on anywhere you would like in the project (Note: You will need the path to this directory while running the script).
```

Comment:
I think "anywhere you would like in the project" can be dropped. It's a bit confusing because I thought it was suggesting I place that directory *within this project*.

Reply:
That was my original intent but I guess it would work as long as the path was correct

Reply:
I don't like to put stuff in my working directory that I have to add to `.gitignore` 😁